src/snowflake/connector/network.py

```diff
@@ -17,6 +17,7 @@
 from threading import Lock
 
 import OpenSSL.SSL
+from urllib3.util.url import parse_url
```

Comment:
Please update this import to use the vendored version, otherwise we need to add back our dependency on `urllib3`